pkgs/top-level/python-packages.nix

```
@@ -5645,6 +5645,8 @@ in {
 
   purl = callPackage ../development/python-modules/purl { };
 
+  push_receiver = callPackage ../development/python-modules/push_receiver { };
```

Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)
```suggestion
  push-receiver = callPackage ../development/python-modules/push-receiver { };
```

---

Thanks, done!